Sources/PackageModel/SwiftSDKs/SwiftSDK.swift

```diff
@@ -738,7 +730,7 @@ public struct SwiftSDK: Equatable {
     /// Appends a path to the array of toolset root paths.
```

Comment:
Nit: This comment needs to be changed to reflect the behavior change.

Reply:
There is also https://github.com/apple/swift-package-manager/blob/main/Sources/PackageModel/SwiftSDKs/SwiftSDK.swift#L717-L719 that needs to be fixed.